examples/nitclk/multi-device-configured-acquisition-tclk.py

```diff
@@ -0,0 +1,294 @@
+            trigger_coupling = niscope_types.TriggerCoupling.TRIGGER_COUPLING_NISCOPE_VAL_DC,
+            holdoff = 0.0,
+            delay = 0.0
+            ))
+            CheckForError(scope_service, vi, trigger_window_result.status)
+        elif trigger_type == 5:
+            trigger_immediate_result = scope_service.ConfigureTriggerImmediate(niscope_types.ConfigureTriggerImmediateRequest(
+            vi = vi
+            ))
+            CheckForError(scope_service, vi, trigger_immediate_result.status)
+
+        if (previous_min_sample_rate != min_sample_rate or previous_max_input_frequency != max_input_frequency or previous_min_record_length != num_records ):
+            # Use NI-TClk to configure appropriate parameters, synchronize digitizers, and initiate operation.
+            homogenous_trigger_result = tclk_service.ConfigureForHomogeneousTriggers(nitclk_types.ConfigureForHomogeneousTriggersRequest(
+                sessions = sessions
+            ))
+            CheckForError(tclk_service, vi, homogenous_trigger_result.status)
+            
+            synchronize_result = tclk_service.Synchronize(nitclk_types.SynchronizeRequest(
+                sessions = sessions,
+                min_tclk_period = 0.0	
+            ))
+            CheckForError(tclk_service, vi, synchronize_result.status)
+        previous_trigger_type = trigger_type
+        previous_min_sample_rate = min_sample_rate
+        previous_max_input_frequency = max_input_frequency
+        previous_min_record_length = num_records
+
+        initiate_result = tclk_service.Initiate(nitclk_types.InitiateRequest(
+                sessions = sessions
+        ))
+        CheckForError(tclk_service, vi, initiate_result.status)
+
+    # Setup a plot to draw the captured waveform
+        fig = plt.gcf()
+        fig.show()
+        fig.canvas.draw()
+
+    # Handle closing of plot window.
+        closed = False
+        def on_close(event):
+            global closed
+            closed = True
+        fig.canvas.mpl_connect('close_event', on_close)
+
+        print("\nReading values in loop. CTRL+C to stop.\n")
+        try:
+            while not closed:
+                # Clear the plot and setup the axis
+                plt.clf()
+                plt.axis([0, 1000, -5, 5])
+                plt.ylabel('Amplitude')
+                plt.xlabel('Samples')
+                # Read a waveform from the scope
+                for session in sessions:
+                    read_result = scope_service.Read(niscope_types.ReadRequest(
+                        vi = session,
+                        channel_list = channels,
+                        timeout = 1,
+                        num_samples = 1000
+                    ))
+                    CheckForError(scope_service, session, read_result.status)
+                    # Update the plot with the new waveform
+                    plt.plot(read_result.waveform[0:1000])
```

Comment:
@Jha002  Before plotting, I think we wanted to round off to some number of decimal places know?